benchmarks/summarize.py

```diff
@@ -57,6 +57,9 @@ def summarize(bench_files):
             'speedup': speedup,
         })
 
+    ## Filter out modes which don't have data, and thus no speedups
+    speedups = {k: v for k, v in speedups.items() if v != []}
```

Comment:
```suggestion
    # Filter out modes which don't have data, and thus no speedups.
    speedups = {k: v for k, v in speedups.items() if v}
```

Empty lists are falsy, so no need for `!= []`.